save-core/src/commonTest/kotlin/org/cqfn/save/core/MergeConfigsTest.kt

```diff
@@ -169,9 +166,6 @@ class MergeConfigsTest {
         val config3 = TestConfig(toml3, config2, mutableListOf(generalConfig3, warnConfig3, fixConfig3))
         val config4 = TestConfig(toml4, config3, mutableListOf(generalConfig4))
 
-        config1.mergeConfigWithParents()
-        config2.mergeConfigWithParents()
-        config3.mergeConfigWithParents()
         config4.mergeConfigWithParents()
```

Comment:
Here we suppose to start from the top config, i.e. from `config1`

Reply:
yes